Makefile

```diff
@@ -206,6 +206,10 @@ chat2: | build deps librln
 	echo -e $(BUILD_MSG) "build/$@" && \
 		$(ENV_SCRIPT) nim chat2 $(NIM_PARAMS) waku.nims
 
+wakustealthcommitments: | build deps librln
```

Comment:
Perhaps remove Make target (and nimble task)?

Reply:
addressed in fea64e05